NonEmergencyAmbulanceTransportation/R4/resources/Questionnaire-R4-NonEmergencyAmbulanceTransportationProgressNote.json

```diff
@@ -0,0 +1,551 @@
+              "linkId": "8.5.2",
+              "text": "Chronic wounds requiring immobilization"
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "code": [],
+          "required": false,
+          "linkId": "8.6",
+          "text": "Attendant required during transport",
+          "item": [
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.1",
+              "text": "Morbid obesity requires additional personnel/equipment to handle"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.2",
+              "text": "Third party attendant required to regulate or adjust oxygen en route"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.3",
+              "text": "Special handling en route – isolation"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.4",
+              "text": "IV medications/fluids required during transport"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.5",
+              "text": "Restraints (physical or chemical) anticipated or used during transport"
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "code": [],
+          "required": false,
+          "linkId": "8.7",
+          "text": "Mental",
+          "item": [
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.7.1",
+              "text": "Danger to self or others"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.7.2",
+              "text": "Confused, combative, lethargic, comatose"
+            }
+          ]
+        },
+        {
+          "type": "boolean",
+          "code": [],
+          "required": false,
+          "linkId": "8.8",
+          "text": "Other"
+        },
+        {
+          "type": "text",
+          "code": [],
+          "required": false,
+          "linkId": "8.9",
+          "text": "Describe",
+          "enableWhen": [
+            {
+              "answerBoolean": true,
+              "question": "8.8",
+              "operator": "="
+            }
+          ]
+        },
+        {
+          "type": "boolean",
+          "code": [],
+          "required": false,
+          "linkId": "8.10",
+          "text": "Are there services that cannot be provided in the patient’s current setting?"
+        },
+        {
+          "type": "group",
```

Comment:
Change this item to "open-choice" and move subitems (except other) to answerOption

Reply:
Please provide example (via slack) of how this has been implemented in another rule set.